tests/functional/firefox/test_notes.py

```diff
@@ -0,0 +1,113 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+import pytest
+
+from pages.firefox.accounts import FirefoxNotesPage
+
+
+@pytest.mark.nondestructive
+def test_firefox_desktop_release_page(base_url, selenium):
+    #firefox/65.0/releasenotes/
+    page = FirefoxNotesPage(selenium, base_url).open()
+    assert page.version is '65.0'
+    assert page.channel is 'Firefox Release'
+    assert page.gtm_id is '/firefox/releasenotes/'
```

Comment:
The gtm page ID is something not really critical to a piece of user functionality on the page, so I'm not sure it really belongs in a functional test like this? You could try and test for this without needing to spin up a browser, using a [python unit test](https://github.com/mozilla/bedrock/blob/master/bedrock/firefox/tests/test_views.py#L791-L809), or failing that a headless test.

Reply:
It's something I had to work really hard to get right in #6823. I suppose a unit test would be more appropriate but I don't know how to unit test logic distributed across 3 templates.

Reply:
Do we have headless tests currently?

Reply:
> Do we have headless tests currently?

Not as far as I'm aware. The unit test I links to is an example of how to request a URL and parse the HTML for something.